src/styles/main.scss

```diff
@@ -0,0 +1,75 @@
+$item-bg-color: #eee;
+$calendar-item-size: 100px;
+$calendar-gap: 1px;
+$calendar-item-width: calc(#{$calendar-item-size} + #{$calendar-gap});
```

Comment:
The `$calendar-item-width` is calculated using `calc` function that combines `$calendar-item-size` and `$calendar-gap`. However, this is not necessary because SCSS can perform arithmetic operations directly on variables. You can simplify this by using arithmetic operations: `$calendar-item-width: $calendar-item-size + $calendar-gap;`.